### PR TITLE
doc(project): added documentation

### LINKED ISSUE #3: repo documentation
Create minimal documentation for the repo

Elements needed:

- description
- pre-required tools
- running the project locally
- how to contribute

### LINKED ISSUE #34: docs: add the new commands to README
# Description
we have added new commands to use the DB and testing/linting, contributors need to know how to use them

# acceptance criteria
- clear steps that you can copy/paste
- following the readme steps, we can launch the app
- following the readme steps, we can run all tests /checks on the app

### PR DESCRIPTION
**Pull request by osminosm**
Created some documentation on how to start working on the project, and contribution rules.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Contribution guide
+
+All development is done through a process using [issues](https://github.com/ioprodz/ioprodz/issues) on this repositories.
```
Comment (nazimboudeffa) [inline CONTRIBUTING.md:3]: I agree with the fact of working on an issue instead of coding without

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+#### Commits
+
+- Commits should be small
+- commit message conventions (TBD)
+
+#### Pull requests
+
+- should be always `rebased` and have no conflicts with main.
+- when done working, Pull request shoud be passed from Draft to Ready, and have reviewers assigned.
+- Once you have 2 approuvals, you can sqash and merge your changes.
```
Comment (nazimboudeffa) [inline CONTRIBUTING.md:39]: Maybe when the project will be bigger

**Review by badido39 — APPROVED** [review]

**Review by nazimboudeffa — APPROVED** [review]
It's okey for me

**Review by raxrs — APPROVED** [review]
well done

**Pull request merged**